api/envoy/config/core/v3/protocol.proto

```diff
@@ -102,6 +102,12 @@ message QuicProtocolOptions {
   // A comma-separated list of strings representing QUIC client connection options defined in
   // `QUICHE <https://github.com/google/quiche/blob/main/quiche/quic/core/crypto/crypto_protocol.h>`_ and to be sent by upstream connections.
   string client_connection_options = 7;
+
+  // The duration that a QUIC connection stays idle before it closes itself. If this field is not present, QUIC default will be applied.
```

Comment:
what is the QUIC default?  is it in the spec?

Reply:
Also if we think the default is fine for like internal quic corporate proxy but we think 30s is better for internet client facing I think we should document both here and in H3 docs.